packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -14,6 +14,7 @@ import { _, closestSupportedLocale } from '@joplin/lib/locale';
 import useContextMenu from './utils/useContextMenu';
 import { copyHtmlToClipboard } from '../../utils/clipboardUtils';
 import shim from '@joplin/lib/shim';
+import Setting from '@joplin/lib/models/Setting';
```

Comment:
We shouldn't make this class depend on the Setting class. In fact I see there's a customCss prop passed to the component. Doesn't it already contain the content of userstyle.css?

Reply:
I see `customCss` does indeed contain the correct styles, but it's not passed to the TinyMCE component (this receives `NoteBodyEditorProps`, not `NoteEditorProps` which does contain `customCss`). Shall I just pass the `customCss`-prop down in stead?

Reply:
Hmm, I see that customCss is passed to the HTML renderer in NoteEditor.tsx, which means TinyMCE should already be using the CSS. So not sure what the issue is, I'd need to run it and debug to make sense of it.

Reply:
I’m not sure what you’re referring to. I changed the PR so that the customCSS-prop is passed to the TinyMCE-component, but feel free to decline this PR if you think this should be solved differently!

Reply:
Great!